documentation/release_notes.markdown

```diff
@@ -7,24 +7,311 @@ canonical: "/puppetdb/latest/release_notes.html"
 3.0.0
 -----
 
-### Upgrading
+### Before Upgrading
```

Comment:
We seemed to have lost the warning about the time it might take for the migration, I think when you rebase you'll see the new warning that I amended this morning - it includes recommendations for disk space as well.